scripts/run_lighthouse_tests.py

```diff
@@ -63,11 +63,15 @@
 
 _PARSER.add_argument(
     '--mode', help='Sets the mode for the lighthouse tests',
-    required=True, choices=['accessibility', 'performance'])
+    required=True,
+    choices=['accessibility', 'performance'])
 
 _PARSER.add_argument(
     '--shard', help='Sets the shard for the lighthouse tests',
     required=True, choices=['1', '2'])
+_PARSER.add_argument(
+    '--skip_build', help='Sets whether do webpack build',
+    required=False, choices=['true', 'false'])
```

Comment:
make this such that the default value if false and if we include the `--skip_build flag` then it's true.

The inclusion of the flag should indicate that it's true rather than the user explicitly saying it's true.

Reply:
done.